docs/DEV.md

```diff
@@ -36,4 +36,25 @@ cd $GOPATH/github.com/inlets/inlets-operator
 go get
 
 go build && ./inlets-operator  --kubeconfig "$(kind get kubeconfig-path --name="kind")" --access-key=$(cat ~/do-access-token) --provider digitalocean
-```
\ No newline at end of file
+```
+
+## Run the Go binary with Scaleway
```

Comment:
Let's not run via the Go binary, but via a Docker image and Kubernetes?

There's a helm chart now which can be used instead to set parameters.

Reply:
Yes, but here we're on the developer documentation. We can keep it right ?

Reply:
Can you give an example with the operator and update the helm chart? Both of these are required for usage IMHO. I'll let this section remain in the README, but you'll need to add the other docs.